explorer/src/routes/+page.svelte

```diff
@@ -1,39 +1,193 @@
 <script>
+  import { Badge, Card } from "$lib/dusk/components";
   import { StatisticsPanel } from "$lib/components";
+  import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableRow,
+  } from "$lib/components/table";
+  import { apiBlocks, apiTransactions } from "$lib/mock-data";
+  import { getRelativeTimeString, middleEllipsis } from "$lib/dusk/string";
+  import { luxToDusk } from "$lib/dusk/currency";
+  import { createValueFormatter } from "$lib/dusk/value";
+
+  const numberFormatter = createValueFormatter("en");
+
+  const blocksData = apiBlocks.data.blocks.slice(0, 15);
```

Comment:
Very minor nitpick, but I think it would be nicer to extract these number params as contestants.